_posts/2013-01-14-aria-landmark-roles.md

```diff
@@ -4,31 +4,74 @@ title: |
+    </p>
+  </dd>
+  <dt>search</dt>
+  <dd>
+    <p>
+      Use on the primary search form. Often, but not always, found within a `banner`.
+    </p>
+  </dd>
+  <dt>main</dt>
+  <dd>
+    <p>
+      Designates the primary content area of the current document. Only one main landmark should be exposed to users at a time.
+    </p>
+  </dd>
+  <dt>navigation</dt>
+  <dd>
+    <p>
+      Used to promote an area as a navigation. Combine with a unique <code>aria-label</code> to provide context of the navigation's purpose. e.g. <code>aria-label="primary"</code>.
+    </p>
+  </dd>
+  <dt>contentinfo</dt>
+  <dd>
+    <p>Typically the "footer" of your page that contains information about the parent document such as copyrights and links to privacy statements.</p>
+  </dd>
+</dl>
 
-    <header role="banner" class="site-header">
+Implementing landmarks in your documents is a straight forward process. Simply add the <code>role</code> attribute referencing the appropriate landmark value. For example:
+```
+<footer role="contentinfo">...</footer>
+```
 
-So, why do you see this warning when you [validate your html](https://validator.w3.org/)?
 
-<p class="info warning"><strong>Warning</strong>: <span>The <code>banner</code> role is unnecessary for element <code>header</code>.</span></p><p class="location">From line <span class="first-line">60</span>, column <span class="first-col">1712</span>; to line <span class="last-line">60</span>, column <span class="last-col">1758</span></p><p class="extract"><code>/a&gt;&lt;/div&gt; <b>&lt;header class="header fixed-pos" role="banner"&gt;</b>&lt;div c</code></p>
+## HTML5 Implicit Mappings of Landmark Roles
+Before you start adding <abbr>ARIA</abbr> roles to your HTML elements, you should be aware that many of these landmarks will be natively conveyed by proper HTML usage. For example, the following markup snippet will produce a warning in modern HTML and accessibility automated checkers:
 
-You see the warning ([consider fixing/changing](https://www.paciellogroup.com/blog/2015/08/short-note-on-html-conformance-checking/)) because, in HTML5, several of the landmark roles are implicit via the native structural elements.
+```
+<header role="banner" class="site-header">...</header>
+```
 
-**HTML5 Implicit Mappings**
+<figure class="figure-callout">
+  <samp>
+    <strong>Warning</strong>: <span>The <code>banner</code> role is unnecessary for element <code>header</code>.</span>
+    <br>
+    <code><b>&lt;header class="site-header" role="banner"&gt;</b>...</code>
+  </samp>
+  <figcaption>
+    A snippet of the warning output from the <a href="https://validator.w3.org/nu/">Nu HTML Validator</a>.
+  </figcaption>
+</figure>
 
-<table class="table table-striped table-bordered">
+The following table outlines the different <abbr>ARIA</abbr> landmarks, and the HTML5 element they are associated with.
```

Comment:
`title`

Reply:
debatable.  title is on the first instance, it doesn't necessarily need to be on all future instances

Reply:
Then remove `abbr`? Without the `title` what's the point?

Reply:
because it's still an abbreviation?